astro/src/content/docs/extend/examples/multi-application-dashboard.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Multi-Application Dashboard
+description: Create a dashboard page with a list of applications a user can log in to.
+navcategory: developer
+section: extend
+subcategory: examples
+---
+
+import Aside from 'src/components/Aside.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import IconButton from 'src/components/icon/Icon.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+
+## Overview
+
+If you're using FusionAuth to manage many applications, you might want to provide your users with a central list of applications, so they know what's available. For example, Google does this for its apps.
+
+![Google app selector](/img/docs/extend/examples/multi-application-dashboard/googleGrid.png)
+
+In this guide, you'll learn to make a similar page — a dashboard linking to all the applications in a FusionAuth tenant.
+
+In the language of authentication, FusionAuth is an identity provider (IdP) and your applications and websites are the service providers (SPs). One way to authenticate a user from a central dashboard is to use an identity-provider-initiated login. In other words, the dashboard will log the user in, and then redirect them to the selected app. In short, a service provider application will receive a login that it didn't initiate itself. This has security complexities and dangers.
+
+This guide will demonstrate a simpler and safer way to authenticate users from a central dashboard by making each application link in the dashboard point to the application's login page. From that point onward, authentication follows the standard OAuth Authorization Code Flow.
+
+The next section of this guide will show you how to make a dashboard for two existing FusionAuth applications in the same tenant. All this requires is customizing the theme for the index landing page of the FusionAuth site. The guide will first show you how to install FusionAuth and make two simple web application web servers. If you already have a FusionAuth installation with existing applications, you can skip ahead to the section that creates the dashboard.
+
+If you want to follow along with the full guide, you will need Docker installed.
+
+## Download The Example Repository And Run FusionAuth
+
+Use `git clone` to clone the repository at https://github.com/fusionauth/fusionauth-example-multiapp-dashboard, or download and unzip it. 
```

Comment:
This repo doesn't exist?

Reply:
When it does get moved over, we should add it to exampleapps.json as well.

Reply:
This is the repo https://github.com/ritza-co/fusionauth-example-multiapp-dashboard

We have added it to exampleapps, assuming you keep the same name when moving it to your org

Reply:
sounds good.This is there now: https://github.com/FusionAuth/fusionauth-example-multiapp-dashboard